packages/types/src/api/web/app/rowAction.ts

```diff
@@ -8,6 +8,7 @@ export interface RowActionResponse extends RowActionData {
   id: string
   tableId: string
   automationId: string
+  allowedViews: string[] | undefined
```

Comment:
```suggestion
  allowedViews?: string[]
```

Reply:
This type is use as type response. Not having the `?` will enforce us to declare this field, even if it is set to undefined. Otherwise, we might be missing some mappings and the type will not complain